packages/app/src/components/VisualizationOptions/Options/DigitGroupSeparator.js

```diff
@@ -1,19 +1,14 @@
 import React from 'react'
-import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
 
 import i18n from '@dhis2/d2-i18n'
 
 import SelectBaseOption from './SelectBaseOption'
 
-import { sGetSettingsDigitGroupSeparator } from '../../../reducers/settings'
-
-const DigitGroupSeparator = ({ defaultValue }) => (
+const DigitGroupSeparator = () => (
     <SelectBaseOption
         label={i18n.t('Digit group separator')}
         option={{
             name: 'digitGroupSeparator',
-            defaultValue,
```

Comment:
Will the system setting still be used as the default selected value in the ui?

Reply:
Yes, because option components are connected to the store and read their value from it.
Default values are used to set the `value` prop on the components.
Now this particular option implementation is back to what it was initially, with not default and the value is just read from the store.
So by setting the default value properly in the store whenever the ui portion of it is reset, we can assure we always have the correct default matching the system settings one.